src/components/global/DialogDefault.vue

```diff
@@ -43,6 +43,10 @@ export default defineComponent({
       type: Boolean,
       default: false,
     },
+    minWidth: {
```

Comment:
Could you update component doc props section, please?

Reply:
Updated.